### ISSUE TITLE
ALT key is (maybe) being eaten by the desktop

### ISSUE THREAD

**Issue by cpicanco** [BunsenLabs/bunsen-configs#48]
Hi,
I just reported this issue: bugs.freepascal.org/view.php?id=30993. I thought it was related with lazarus. It may be related to the open box desktop configuration. Key combinations with ALT are (maybe) being eaten by the desktop. For example, CTRL+ALT+O should open the debug window when Lazarus is open. It does not, and as the [pascal issue discussion](bugs.freepascal.org/view.php?id=30993) suggests, it is not related to Lazarus. Please, could you provide some feedback on this?

**Comment by ghost** [comment]
There is no keybinding configured to C-A-o in Openbox; could you please
run the 'xev' command from a terminal, focus the white window, press the
keychords

  Alt-o
  Ctrl-o
  Ctrl-Alt-o

and copy&paste the resulting output in the terminal here?

**Comment by cpicanco** [comment]
> alt-o

```
~$ xev
Outer window is 0x3200001, inner window is 0x3200002

PropertyNotify event, serial 8, synthetic NO, window 0x3200001,
    atom 0x27 (WM_NAME), time 233796107, state PropertyNewValue

PropertyNotify event, serial 9, synthetic NO, window 0x3200001,
    atom 0x22 (WM_COMMAND), time 233796107, state PropertyNewValue

PropertyNotify event, serial 10, synthetic NO, window 0x3200001,
    atom 0x28 (WM_NORMAL_HINTS), time 233796107, state PropertyNewValue

CreateNotify event, serial 11, synthetic NO, window 0x3200001,
    parent 0x3200001, window 0x3200002, (10,10), width 50, height 50
border_width 4, override NO

PropertyNotify event, serial 14, synthetic NO, window 0x3200001,
    atom 0x121 (WM_PROTOCOLS), time 233796107, state PropertyNewValue

MapNotify event, serial 15, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200002, override NO

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x140 (_NET_WM_VISIBLE_NAME), time 233796108, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x142 (_NET_WM_VISIBLE_ICON_NAME), time 233796108, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17b (_OB_APP_ROLE), time 233796108, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17d (_OB_APP_NAME), time 233796108, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17e (_OB_APP_CLASS), time 233796108, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17f (_OB_APP_GROUP_NAME), time 233796108, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x180 (_OB_APP_GROUP_CLASS), time 233796108, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17c (_OB_APP_TITLE), time 233796108, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x181 (_OB_APP_TYPE), time 233796108, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x148 (_NET_WM_ICON), time 233796108, state PropertyNewValue

ReparentNotify event, serial 18, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, parent 0xc50d8f,
    (0,0), override NO

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x145 (_NET_WM_STATE), time 233796109, state PropertyNewValue

ConfigureNotify event, serial 18, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, (0,0), width 178, height 178,
    border_width 0, above 0xc50dbb, override NO

ConfigureNotify event, serial 18, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, (1,26), width 178, height 178,
    border_width 0, above 0xc50dbb, override NO

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x14f (_NET_FRAME_EXTENTS), time 233796110, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x14e (_KDE_NET_WM_FRAME_STRUT), time 233796110, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x14b (_NET_WM_ALLOWED_ACTIONS), time 233796110, state PropertyNewValue

ConfigureNotify event, serial 18, synthetic YES, window 0x3200001,
    event 0x3200001, window 0x3200001, (-1,53), width 178, height 178,
    border_width 2, above 0x0, override NO

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x143 (_NET_WM_DESKTOP), time 233796110, state PropertyNewValue

MapNotify event, serial 27, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, override NO

VisibilityNotify event, serial 27, synthetic NO, window 0x3200001,
    state VisibilityUnobscured

Expose event, serial 27, synthetic NO, window 0x3200001,
    (0,0), width 178, height 10, count 3

Expose event, serial 27, synthetic NO, window 0x3200001,
    (0,10), width 10, height 58, count 2

Expose event, serial 27, synthetic NO, window 0x3200001,
    (68,10), width 110, height 58, count 1

Expose event, serial 27, synthetic NO, window 0x3200001,
    (0,68), width 178, height 110, count 0

PropertyNotify event, serial 27, synthetic NO, window 0x3200001,
    atom 0x122 (WM_STATE), time 233796113, state PropertyNewValue

FocusIn event, serial 27, synthetic NO, window 0x3200001,
    mode NotifyNormal, detail NotifyNonlinear

KeymapNotify event, serial 27, synthetic NO, window 0x0,
    keys:  4294967229 0   0   0   16  0   0   0   0   0   0   0   0   0   0   0   
           0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   

PropertyNotify event, serial 28, synthetic NO, window 0x3200001,
    atom 0x149 (_NET_WM_ICON_GEOMETRY), time 233796119, state PropertyNewValue

KeyRelease event, serial 45, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233796142, (988,122), root:(989,177),
    state 0x0, keycode 36 (keysym 0xff0d, Return), same_screen YES,
"   XLookupString gives 1 bytes: (0d) "
    XFilterEvent returns: False

KeyPress event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233804250, (120,-22), root:(121,33),
    state 0x0, keycode 64 (keysym 0xffe9, Alt_L), same_screen YES,
    XLookupString gives 0 bytes: 
    XmbLookupString gives 0 bytes: 
    XFilterEvent returns: False

KeyPress event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233804575, (120,-22), root:(121,33),
    state 0x8, keycode 32 (keysym 0x6f, o), same_screen YES,
    XLookupString gives 1 bytes: (6f) "o"
    XmbLookupString gives 1 bytes: (6f) "o"
    XFilterEvent returns: False

KeyRelease event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233804830, (120,-22), root:(121,33),
    state 0x8, keycode 32 (keysym 0x6f, o), same_screen YES,
    XLookupString gives 1 bytes: (6f) "o"
    XFilterEvent returns: False

KeyPress event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233804830, (120,-22), root:(121,33),
    state 0x8, keycode 32 (keysym 0x6f, o), same_screen YES,
    XLookupString gives 1 bytes: (6f) "o"
    XmbLookupString gives 1 bytes: (6f) "o"
    XFilterEvent returns: False

KeyRelease event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233804868, (120,-22), root:(121,33),
    state 0x8, keycode 32 (keysym 0x6f, o), same_screen YES,
    XLookupString gives 1 bytes: (6f) "o"
    XFilterEvent returns: False

KeyRelease event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233805379, (120,-22), root:(121,33),
    state 0x8, keycode 64 (keysym 0xffe9, Alt_L), same_screen YES,
    XLookupString gives 0 bytes: 
    XFilterEvent returns: False

ClientMessage event, serial 48, synthetic YES, window 0x3200001,
    message_type 0x121 (WM_PROTOCOLS), format 32, message 0x124 (WM_DELETE_WINDOW)
```

> ctrl-o

```
~$ xev
Outer window is 0x3200001, inner window is 0x3200002

PropertyNotify event, serial 8, synthetic NO, window 0x3200001,
    atom 0x27 (WM_NAME), time 233967757, state PropertyNewValue

PropertyNotify event, serial 9, synthetic NO, window 0x3200001,
    atom 0x22 (WM_COMMAND), time 233967757, state PropertyNewValue

PropertyNotify event, serial 10, synthetic NO, window 0x3200001,
    atom 0x28 (WM_NORMAL_HINTS), time 233967757, state PropertyNewValue

CreateNotify event, serial 11, synthetic NO, window 0x3200001,
    parent 0x3200001, window 0x3200002, (10,10), width 50, height 50
border_width 4, override NO

PropertyNotify event, serial 14, synthetic NO, window 0x3200001,
    atom 0x121 (WM_PROTOCOLS), time 233967758, state PropertyNewValue

MapNotify event, serial 15, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200002, override NO

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x140 (_NET_WM_VISIBLE_NAME), time 233967758, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x142 (_NET_WM_VISIBLE_ICON_NAME), time 233967758, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17b (_OB_APP_ROLE), time 233967758, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17d (_OB_APP_NAME), time 233967758, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17e (_OB_APP_CLASS), time 233967758, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17f (_OB_APP_GROUP_NAME), time 233967758, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x180 (_OB_APP_GROUP_CLASS), time 233967758, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17c (_OB_APP_TITLE), time 233967758, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x181 (_OB_APP_TYPE), time 233967758, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x148 (_NET_WM_ICON), time 233967759, state PropertyNewValue

ReparentNotify event, serial 18, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, parent 0xc5165d,
    (0,0), override NO

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x145 (_NET_WM_STATE), time 233967759, state PropertyNewValue

ConfigureNotify event, serial 20, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, (0,0), width 178, height 178,
    border_width 0, above 0xc51689, override NO

ConfigureNotify event, serial 20, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, (1,26), width 178, height 178,
    border_width 0, above 0xc51689, override NO

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x14f (_NET_FRAME_EXTENTS), time 233967760, state PropertyNewValue

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x14e (_KDE_NET_WM_FRAME_STRUT), time 233967760, state PropertyNewValue

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x14b (_NET_WM_ALLOWED_ACTIONS), time 233967760, state PropertyNewValue

ConfigureNotify event, serial 20, synthetic YES, window 0x3200001,
    event 0x3200001, window 0x3200001, (-1,53), width 178, height 178,
    border_width 2, above 0x0, override NO

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x143 (_NET_WM_DESKTOP), time 233967760, state PropertyNewValue

MapNotify event, serial 28, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, override NO

VisibilityNotify event, serial 28, synthetic NO, window 0x3200001,
    state VisibilityUnobscured

Expose event, serial 28, synthetic NO, window 0x3200001,
    (0,0), width 178, height 10, count 3

Expose event, serial 28, synthetic NO, window 0x3200001,
    (0,10), width 10, height 58, count 2

Expose event, serial 28, synthetic NO, window 0x3200001,
    (68,10), width 110, height 58, count 1

Expose event, serial 28, synthetic NO, window 0x3200001,
    (0,68), width 178, height 110, count 0

PropertyNotify event, serial 28, synthetic NO, window 0x3200001,
    atom 0x122 (WM_STATE), time 233967764, state PropertyNewValue

FocusIn event, serial 28, synthetic NO, window 0x3200001,
    mode NotifyNormal, detail NotifyNonlinear

KeymapNotify event, serial 28, synthetic NO, window 0x0,
    keys:  4294967228 0   0   0   16  0   0   0   0   0   0   0   0   0   0   0   
           0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   

PropertyNotify event, serial 30, synthetic NO, window 0x3200001,
    atom 0x149 (_NET_WM_ICON_GEOMETRY), time 233967769, state PropertyNewValue

KeyRelease event, serial 42, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233967806, (856,164), root:(857,219),
    state 0x0, keycode 36 (keysym 0xff0d, Return), same_screen YES,
"   XLookupString gives 1 bytes: (0d) "
    XFilterEvent returns: False

KeyPress event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233975531, (101,-20), root:(102,35),
    state 0x0, keycode 37 (keysym 0xffe3, Control_L), same_screen YES,
    XLookupString gives 0 bytes: 
    XmbLookupString gives 0 bytes: 
    XFilterEvent returns: False

KeyPress event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233975744, (101,-20), root:(102,35),
    state 0x4, keycode 32 (keysym 0x6f, o), same_screen YES,
    XLookupString gives 1 bytes: (0f) ""
    XmbLookupString gives 1 bytes: (0f) ""
    XFilterEvent returns: False

KeyRelease event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233975840, (101,-20), root:(102,35),
    state 0x4, keycode 32 (keysym 0x6f, o), same_screen YES,
    XLookupString gives 1 bytes: (0f) ""
    XFilterEvent returns: False

KeyRelease event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 233976091, (101,-20), root:(102,35),
    state 0x4, keycode 37 (keysym 0xffe3, Control_L), same_screen YES,
    XLookupString gives 0 bytes: 
    XFilterEvent returns: False

ClientMessage event, serial 48, synthetic YES, window 0x3200001,
    message_type 0x121 (WM_PROTOCOLS), format 32, message 0x124 (WM_DELETE_WINDOW)

```

> ctrl+alt+o

```
~$ xev
Outer window is 0x3200001, inner window is 0x3200002

PropertyNotify event, serial 8, synthetic NO, window 0x3200001,
    atom 0x27 (WM_NAME), time 234059622, state PropertyNewValue

PropertyNotify event, serial 9, synthetic NO, window 0x3200001,
    atom 0x22 (WM_COMMAND), time 234059622, state PropertyNewValue

PropertyNotify event, serial 10, synthetic NO, window 0x3200001,
    atom 0x28 (WM_NORMAL_HINTS), time 234059622, state PropertyNewValue

CreateNotify event, serial 11, synthetic NO, window 0x3200001,
    parent 0x3200001, window 0x3200002, (10,10), width 50, height 50
border_width 4, override NO

PropertyNotify event, serial 14, synthetic NO, window 0x3200001,
    atom 0x121 (WM_PROTOCOLS), time 234059622, state PropertyNewValue

MapNotify event, serial 15, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200002, override NO

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x140 (_NET_WM_VISIBLE_NAME), time 234059623, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x142 (_NET_WM_VISIBLE_ICON_NAME), time 234059623, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17b (_OB_APP_ROLE), time 234059623, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17d (_OB_APP_NAME), time 234059623, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17e (_OB_APP_CLASS), time 234059623, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17f (_OB_APP_GROUP_NAME), time 234059623, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x180 (_OB_APP_GROUP_CLASS), time 234059623, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x17c (_OB_APP_TITLE), time 234059623, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x181 (_OB_APP_TYPE), time 234059623, state PropertyNewValue

PropertyNotify event, serial 18, synthetic NO, window 0x3200001,
    atom 0x148 (_NET_WM_ICON), time 234059623, state PropertyNewValue

ReparentNotify event, serial 18, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, parent 0xc518df,
    (0,0), override NO

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x145 (_NET_WM_STATE), time 234059624, state PropertyNewValue

ConfigureNotify event, serial 20, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, (0,0), width 178, height 178,
    border_width 0, above 0xc5190b, override NO

ConfigureNotify event, serial 20, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, (1,26), width 178, height 178,
    border_width 0, above 0xc5190b, override NO

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x14f (_NET_FRAME_EXTENTS), time 234059624, state PropertyNewValue

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x14e (_KDE_NET_WM_FRAME_STRUT), time 234059624, state PropertyNewValue

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x14b (_NET_WM_ALLOWED_ACTIONS), time 234059625, state PropertyNewValue

ConfigureNotify event, serial 20, synthetic YES, window 0x3200001,
    event 0x3200001, window 0x3200001, (659,53), width 178, height 178,
    border_width 2, above 0x0, override NO

PropertyNotify event, serial 20, synthetic NO, window 0x3200001,
    atom 0x143 (_NET_WM_DESKTOP), time 234059625, state PropertyNewValue

MapNotify event, serial 38, synthetic NO, window 0x3200001,
    event 0x3200001, window 0x3200001, override NO

VisibilityNotify event, serial 38, synthetic NO, window 0x3200001,
    state VisibilityUnobscured

Expose event, serial 38, synthetic NO, window 0x3200001,
    (0,0), width 178, height 10, count 3

Expose event, serial 38, synthetic NO, window 0x3200001,
    (0,10), width 10, height 58, count 2

Expose event, serial 38, synthetic NO, window 0x3200001,
    (68,10), width 110, height 58, count 1

Expose event, serial 38, synthetic NO, window 0x3200001,
    (0,68), width 178, height 110, count 0

PropertyNotify event, serial 38, synthetic NO, window 0x3200001,
    atom 0x122 (WM_STATE), time 234059629, state PropertyNewValue

FocusIn event, serial 38, synthetic NO, window 0x3200001,
    mode NotifyNormal, detail NotifyNonlinear

KeymapNotify event, serial 38, synthetic NO, window 0x0,
    keys:  125 0   0   0   16  0   0   0   0   0   0   0   0   0   0   0   
           0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   

PropertyNotify event, serial 38, synthetic NO, window 0x3200001,
    atom 0x149 (_NET_WM_ICON_GEOMETRY), time 234059634, state PropertyNewValue

KeyRelease event, serial 42, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 234059673, (-496,178), root:(165,233),
    state 0x0, keycode 36 (keysym 0xff0d, Return), same_screen YES,
"   XLookupString gives 1 bytes: (0d) "
    XFilterEvent returns: False

KeyPress event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 234064588, (100,-20), root:(761,35),
    state 0x0, keycode 37 (keysym 0xffe3, Control_L), same_screen YES,
    XLookupString gives 0 bytes: 
    XmbLookupString gives 0 bytes: 
    XFilterEvent returns: False

KeyPress event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 234064716, (100,-20), root:(761,35),
    state 0x4, keycode 64 (keysym 0xffe9, Alt_L), same_screen YES,
    XLookupString gives 0 bytes: 
    XmbLookupString gives 0 bytes: 
    XFilterEvent returns: False

FocusOut event, serial 48, synthetic NO, window 0x3200001,
    mode NotifyGrab, detail NotifyAncestor

FocusIn event, serial 48, synthetic NO, window 0x3200001,
    mode NotifyUngrab, detail NotifyAncestor

KeymapNotify event, serial 48, synthetic NO, window 0x0,
    keys:  2   0   0   0   32  0   0   0   1   0   0   0   0   0   0   0   
           0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   

KeyRelease event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 234065222, (100,-20), root:(761,35),
    state 0xc, keycode 64 (keysym 0xffe9, Alt_L), same_screen YES,
    XLookupString gives 0 bytes: 
    XFilterEvent returns: False

KeyRelease event, serial 48, synthetic NO, window 0x3200001,
    root 0x16d, subw 0x0, time 234065248, (100,-20), root:(761,35),
    state 0x4, keycode 37 (keysym 0xffe3, Control_L), same_screen YES,
    XLookupString gives 0 bytes: 
    XFilterEvent returns: False

ClientMessage event, serial 48, synthetic YES, window 0x3200001,
    message_type 0x121 (WM_PROTOCOLS), format 32, message 0x124 (WM_DELETE_WINDOW)
```

**Comment by ghost** [comment]
Thanks. It looks like Ctrl-Alt-o is indeed being intercepted, namely by
the clipit process (clipboard manager). Change the clipit hotkey by
clicking on the clipboard icon in the tray, then go to the preference
panel, then configure a different "Offline mode hotkey" in the hotkey
tab. This should fix your problem.

Closes #48.

**Comment by cpicanco** [comment]
Closing clipit resolved the problem, indeed!
